browser-tests/content-regression-tests/tests/resources-learning/search.spec.ts

```diff
@@ -1,11 +1,11 @@
 import { test, expect } from '@playwright/test'
 
-test.describe.skip('Resources and learning', () => {
+test.describe('Resources and learning', () => {
```

Comment:
Need to add the skip to the whole set of tests, it's only been added to a single test here

Reply:
Corrected the skip (now on the original test)